binaries/build_wheels.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A script that downloads ffmpeg, ffprobe, and packager static builds for all
+the platforms we build for and then builds distribution wheels for them.
+"""
+
+import os
+import sys
+import urllib.request
+import setuptools # type: ignore
+import setuptools.command.build_py #type: ignore
+import streamer_binaries
+
+
+# Version constants.
+# Change to download different versions.
+FFMPEG_VERSION = 'n4.4-1'
+PACKAGER_VERSION = 'v2.99.5'
+
+# A map of postfixes that will be combined with the binary download links
+# to achieve a full download link.  Different postfix for each platform.
+# Extend this dictionary to add more platforms.
+PLATFORM_POSTFIXES = {
+    # 64-bit Windows
+    'win_amd64': '-win-x64.exe',
+    # 64-bit Linux
+    'manylinux1_x86_64': '-linux-x64',
+    # Linux on ARM
+    'manylinux2014_aarch64': '-linux-arm64',
+    # 64-bit with 10.9 SDK
+    'macosx_10_9_x86_64': '-osx-x64',
+  }
+
+FFMPEG_DL_PREFIX = 'https://github.com/joeyparrish/static-ffmpeg-binaries/releases/download/' + FFMPEG_VERSION
+PACKAGER_DL_PREFIX = 'https://github.com/joeyparrish/shaka-packager/releases/download/' + PACKAGER_VERSION
+
+# The download links to each binary.  These download links
+# aren't complete, they miss the platfrom-specific postfix.
+BINARIES_DL = [
+    FFMPEG_DL_PREFIX + '/ffmpeg',
+    FFMPEG_DL_PREFIX + '/ffprobe',
+    PACKAGER_DL_PREFIX + '/packager',
+  ]
```

Comment:
style nit: align with `BINARIES_DL`

Reply:
I don't get it.  Do you mean i should remove all the indentation?

Reply:
got it, my bad.